components/Navbar.js

```diff
@@ -63,17 +67,26 @@ const NavLinks = ({ openMenu, components, onClick }) => {
 
 export function Navbar() {
     const [openMenu, toggleMenu] = useState(false);
+    const handleClick = (state) => {
+        //feels hacky, like maybe the state should be managed higher up?
```

Comment:
I was just reading an [article about the framer-motion page transitions](https://wallis.dev/blog/nextjs-page-transitions-with-framer-motion) that I think had a handle for this. Scroll down to the part titled "Solving the scroll on link change issue." They recommended creating a link component called `NoScrollLink` that uses the [scroll="false"](https://nextjs.org/docs/api-reference/next/link#disable-scrolling-to-the-top-of-the-page) attribute.

Would that work here?

Reply:
I'll read that and report back tonight or tomorrow

Reply:
I don't think the handleClick method is tackling the same problem as the the NoScrollLink component is tackling.

The NoScrollLink component seems to be handling the on click event which scrolls to the top when the route is changed and then triggers the animation. This causes the awkward motion and stutter/judder and feels janky. I do believe this is a useful component regardless and is probably a good idea to implement the component and have our styled navigation links extend the NoScrollLink component.

The method in question is preventing scrolling on the background elements (header, main, footer) on mobile when the mobile menu is open. If a user were to open a menu and scroll down the navigation links to see the newsletter signup and decided to close the menu, they would be in a different position on the page as they would be scrolling both the open mobile menu and the page itself.

In hindsight, not even sure how big of deal it is